GTBitsOfGood/hope-sustains-life
# Create Password Reset Page

## Create Password Reset Page

Issue Number(s): #164 

What does this PR change and why?
* Allows users to change their passwords in case they forget their old passwords

### Checklist


### Critical Changes
- `_app.js` prevents rendering headers for `admin` routes. This PR updates that logic to not show the headers for `resetpassword` page as well

### Related PRs

#163 
#162 

### How to Test

1. Navigate to `localhost:3000/resetpassword/[inser random token]`
2. Enter password fields
    * Input field should prevent submitting on empty password fields
    * Form should display error msg when users make a request with different values for two fields
3. Verify that Loading message is displayed and removed
    * Test this via wrapping `fetch` in `settimeout`
4. Verify that Submit Success or Submit Error messages are displayed depending on the response status code
    * Test this via replacing the fetch url to a valid API endpoint (#163 )
5. Profit lolz
